Lib/distutils/tests/test_check.py

```diff
@@ -99,6 +104,12 @@ def test_check_restructuredtext(self):
         cmd = self._run(metadata, strict=1, restructuredtext=1)
         self.assertEqual(cmd._warnings, 0)
 
+        # check that includes work to test #31292
+        os.chdir(HERE)
```

Comment:
I’m afraid changing the environment like this is not test best practice.

`self._run` should do it, using the helpers in test.support that make sure to restore the previous state.

Reply:
Hm, I just did it manually as I didn’t find any helper in `support` that did what I want.

Reply:
All right then!